core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchReaderIterator.java

```diff
@@ -670,8 +676,11 @@ static void doLookup(ClientContext context, String server, Map<KeyExtent,List<Ra
         TabletType ttype = TabletType.type(requested.keySet());
         boolean waitForWrites = !ThriftScanner.serversWaitedForWrites.get(ttype).contains(server);
 
-        Map<TKeyExtent,List<TRange>> thriftTabletRanges = Translator.translate(requested,
-            Translators.KET, new Translator.ListTranslator<>(Translators.RT));
+       Map<TKeyExtent, List<TRange>> thriftTabletRanges = requested.entrySet().stream()
```

Comment:
With these changes, is it possible to remove any of the constants like `Translators.KET`?